src/tests/container.spec.ts

```diff
@@ -449,6 +464,14 @@ describe('Container', () => {
 
             expect(instance1).not.to.be.equal(instance2);
         });
+
+        it('should register itself for injection', () => {
+            // arrange, act
+            const actual = container.resolve(TContainer);
+            // assert
```

Comment:
You can't deny that comments like this one are obvious. Clearly named variables and methods won't need any comments above them. Please remove them.